### DISCUSSION TITLE
(Feature request] Bluetooth guest access to node

### DISCUSSION THREAD

**Discussion by zl3ag** [meshtastic/firmware#5071] [Ideas]
We have a node at our local ham club for people to use.

It would be nice if a guest access mode was available via BT so they can connect via phone app but not fiddle with the node's config. Or maybe password access to the config area?

**Comment by b8b8** [comment]
Use 123456 as the Bluetooth password and post a copy nearby. Turn on "managed" mode so changes to the device can only be made over the admin channel.

**Reply by mtsdawson** [reply]
Just to better understand this, am I correct that with managed mode enabled, the node would still reveal the admin public key to the user? So, hypothetically, the guest user could copy the admin public key, then connect to the node via admin channel when managed mode is enabled?

Or is radio config completely disabled with managed mode enabled?

**Reply by b8b8** [reply]
That's not how public/private encryption works. Copying the admin public key wouldn't help as that is used for decrypting the admin message, not encrypting it. The guest user would need to copy the admin private key that then is decrypted by the public key and allows the admin message to be understood. There is no danger in sharing public keys, they just allow the other person to decode messages encrypted using the private key. 

But to answer your last question: 
Yes the radio config is completely **disabled**, you cannot click on it to even see the sub-list, let alone the actual settings. The Channel page with the QR code is also grayed out and cannot be changed (though the QR code is still visible).
I just confirmed this with Android.

**Comment by mtsdawson** [comment]
Super helpful, thank you. I was aware public is generated by private but just wanted to confirm this didn't have any glaring weaknesses before applying.
________________________________
